libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/ClientManager.java

```diff
@@ -166,40 +164,39 @@ public RestClient peekRestClient(Account acc) {
             SalesforceSDKLogger.i(TAG, "User is logging out", e);
             throw e;
         }
-        String passcodeHash = (SalesforceSDKManager.getInstance().getIsTestRun() ? loginOptions.passcodeHash : SalesforceSDKManager.getInstance().getPasscodeHash());
-        String authToken = SalesforceSDKManager.decryptWithPasscode(accountManager.getUserData(acc, AccountManager.KEY_AUTHTOKEN), passcodeHash);
-        String refreshToken = SalesforceSDKManager.decryptWithPasscode(accountManager.getPassword(acc), passcodeHash);
+        String authToken = SalesforceSDKManager.decrypt(accountManager.getUserData(acc, AccountManager.KEY_AUTHTOKEN));
```

Comment:
Replaced all `decryptWithPasscode...` methods in this file with `decrypt...` and `encryptWithPasscode...` methods with `encrypt...`.